charts/homepage/values.yaml

```diff
@@ -127,6 +139,10 @@ config:
 # F) - name: TZ
 #      value: '{{ .Release.Name }}'
 env:
+  - name: HOMEPAGE_ALLOWED_HOSTS
```

Comment:
I feel like this kind of change would be a major change i.e.: 2.x.x --> 3.x.x bump. I was using a version of `~2` where this supported `HOMEPAGE_ALLOWED_HOSTS: "..."`

Reply:
I swear I tested that. Are you saying it doesn't work now? The base template library supports specifying them in either style.

Reply:
Yeah. Unfortunately I hit an error while running `helm template`. Something about mapping not allowed, expected a sequence